### PR TITLE
Pair work(Cynthia Hadi) Assignment1

### PR DESCRIPTION
**Pull request by HadiM2**

### CODE REVIEW

**Comment by Aramirez61** [comment]
I will review this.

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs [hunk 1]
```diff
@@ -21,12 +21,15 @@ public static void Main(string[] args)
 
     public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
     {
+        Console.WriteLine("Number of Correct Answers: " + numberCorrectAnswers + " out of " + numberOfQuestions);
```
Comment (Aramirez61) [inline PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:24]: Great inclusion of providing an interface for the user to see their score.

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs [hunk 2]
```diff
@@ -1,92 +1,115 @@
...
+        string userGuess = GetGuessFromUser();
+        return DisplayResult(userGuess, question);
+    }
+
+    public static string GetGuessFromUser()
+    {
+        return Console.ReadLine();
+    }
+
+    //public static bool AreYouSure(String answer)
```
Comment (Aramirez61) [inline PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:48]: Is this feature meant to be commented out?

File: PrincessBrideTrivia/PrincessBrideTrivia.Tests/ProgramTests.cs [hunk 3]
```diff
@@ -29,7 +29,7 @@ public void LoadQuestions_RetrievesQuestionsFromFile()
     [TestMethod]
     [DataRow("1", true)]
     [DataRow("2", false)]
-    public void DisplayResult_ReturnsTrueIfCorrect(string userGuess, bool expectedResult)
+    public void DisplayResultReturnsTrueIfCorrect(string userGuess, bool expectedResult)
```
Comment (Aramirez61) [inline PrincessBrideTrivia/PrincessBrideTrivia.Tests/ProgramTests.cs:32]: Remember for tests to follow the naming format of MethodTested_Condition_ExpectedResult

**Review by Aramirez61 — COMMENTED** [review]
Pull Request title includes "Assignment 1" in the title ✔
Partner has also made a commit ✔
Pull Request targets Assignment 1 branch ✔
Issue 1: Application no longer crashes ✔
Issue 2: Unit test properly passes ✔
Extra Credit
Come up with a new feature for the application
Issue 3: Add a new feature request implemented ✔
Issue 3: Feature request unit tested ❌

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs (hunk 2 above)
Comment (HadiM2) [reply PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:48]: Yes, it was, didn't get the test passing in time.

File: PrincessBrideTrivia/PrincessBrideTrivia.Tests/ProgramTests.cs (hunk 3 above)
Comment (HadiM2) [reply PrincessBrideTrivia/PrincessBrideTrivia.Tests/ProgramTests.cs:32]: Thank you for the reminder.

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs (hunk 1 above)
Comment (HadiM2) [reply PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:24]: Thank you, it made sense to add it.

File: PrincessBrideTrivia/PrincessBrideTrivia.Tests/ProgramTests.cs [hunk 4]
```diff
@@ -29,7 +29,7 @@ public void LoadQuestions_RetrievesQuestionsFromFile()
     [TestMethod]
     [DataRow("1", true)]
     [DataRow("2", false)]
-    public void DisplayResult_ReturnsTrueIfCorrect(string userGuess, bool expectedResult)
+    public void DisplayResult_ReturnsTrue_IfCorrect(string userGuess, bool expectedResult)
```
Comment (BenjaminMichaelis) [inline PrincessBrideTrivia/PrincessBrideTrivia.Tests/ProgramTests.cs:32]:
```suggestion
    public void DisplayResult_IfCorrect_ReturnsTrue(string userGuess, bool expectedResult)
```
If adjusting this, I would go one step farther so this actually follows guidelines of SUT_Condition_Result

**Review by BenjaminMichaelis — COMMENTED** [review]
Remember to EXPLICITLY mention your partners in your PR Description. 

Ex:

@HadiM2 @montalcy

**Pull request closed without merging**